data/settings.htm

```diff
@@ -132,7 +132,7 @@ <h5 class="card-header">
 							</div>
 						</div>
 						<div class="row col-sm-12 justify-content-center">
-							<button type="submit" class="btn btn-primary">Update</button>
+							<button type="submit" id="submitSettings" class="btn btn-primary">Update</button>
```

Comment:
As I recall, you can't/shouldn't have multiple elements with the same ID. I see how this is implemented in calibration.js - I'm wondering if the better solution would be to assign all the submit buttons a class specific to them, and then [update the style on the class](https://www.sitepoint.com/change-css-jquery/) rather than the  element ID directly.

Reply:
Resolved with 0ed7a21.